test/distributed/test_pg_wrapper.py

```diff
@@ -33,6 +33,14 @@ def setUp(self):
 
     def _validate_error(self, exception, op_type, rank, tensor):
         err = str(exception)
+        # Test if sequence number is attached
+        seq_num_msg = "The sequence number is: "
+        seq_num_msg_index = err.find(seq_num_msg)
+        seq_num_msg_len = len(seq_num_msg)
+        self.assertTrue(seq_num_msg_index != -1, "Sequence number is missing.")
+        if seq_num_msg_index != -1:
```

Comment:
nit; I think this check is unneeded since if `seq_num_msg_index == -1`, then the previous assert will have already failed, in which case the test does not continue.

Reply:
Done. Thank you.